packages/js-utils/README.md

```diff
@@ -2,7 +2,7 @@
 <p>
   <img alt="Version" src="https://img.shields.io/badge/version-0.1.0-blue.svg?cacheSeconds=2592000" />
   <a href="#" target="_blank">
-    <img alt="License: (Apache)" src="https://img.shields.io/badge/License-Apache_2.0-yellow.svg" />
+    <img alt="License: ISC" src="https://img.shields.io/badge/License-ISC-yellow.svg" />
```

Comment:
this should be apache 2

Reply:
this change and other changes to the readme came after running the jsdoc2md node script.

Reply:
windows issue maybe?

Reply:
most of them seem to be line-breakes being replaced by some characters.
Could you please run the script on the branch on your machine?